--- a/replacePatterns/01-before.java
+++ b/replacePatterns/01-after.java
@@ -1,20 +1,20 @@
 import org.eclipse.collections.api.list.MutableList;

 class Test {
 	void test(MutableList<String> list) {
-		boolean negatedNoneSatisfy = !list.noneSatisfy(s -> s.length() > 5);
-		boolean negatedAnySatisfy = !list.anySatisfy(s -> s.length() > 5);
-		boolean negatedNoneSatisfyMethodRef = !list.noneSatisfy(String::isEmpty);
-		boolean negatedAnySatisfyMethodRef = !list.anySatisfy(String::isEmpty);
+		boolean negatedNoneSatisfy = list.anySatisfy(s -> s.length() > 5);
+		boolean negatedAnySatisfy = list.noneSatisfy(s -> s.length() > 5);
+		boolean negatedNoneSatisfyMethodRef = list.anySatisfy(String::isEmpty);
+		boolean negatedAnySatisfyMethodRef = list.noneSatisfy(String::isEmpty);

-		if (!list.noneSatisfy(s -> s.isEmpty())) {
+		if (list.anySatisfy(s -> s.isEmpty())) {
 			this.doWork();
 		}

-		if (!list.anySatisfy(s -> s.startsWith("a"))) {
+		if (list.noneSatisfy(s -> s.startsWith("a"))) {
 			this.doWork();
 		}
 	}

 	void doWork() {}
 }
