--- a/replacePatterns/01-before.java
+++ b/replacePatterns/01-after.java
@@ -1,10 +1,10 @@
 import org.eclipse.collections.api.list.MutableList;

 class Test {
 	void example(MutableList<String> list) {
-		boolean detectOptionalIsPresent = list.detectOptional(s -> s.length() > 5).isPresent();
-		boolean negatedDetectOptionalIsPresent = !list.detectOptional(s -> s.length() > 5).isPresent();
-		boolean detectOptionalIsEmpty = list.detectOptional(s -> s.length() > 5).isEmpty();
-		boolean negatedDetectOptionalIsEmpty = !list.detectOptional(s -> s.length() > 5).isEmpty();
+		boolean detectOptionalIsPresent = list.anySatisfy(s -> s.length() > 5);
+		boolean negatedDetectOptionalIsPresent = list.noneSatisfy(s -> s.length() > 5);
+		boolean detectOptionalIsEmpty = list.noneSatisfy(s -> s.length() > 5);
+		boolean negatedDetectOptionalIsEmpty = list.anySatisfy(s -> s.length() > 5);
 	}
 }
