--- a/replacePatterns/02-before.java
+++ b/replacePatterns/02-after.java
@@ -1,10 +1,10 @@
 import org.eclipse.collections.api.block.predicate.Predicate;
 import org.eclipse.collections.api.list.ImmutableList;

 class TestImmutable {
 	boolean test(
 			ImmutableList<String> list,
 			Predicate<String> predicate) {
-		return list.select(predicate).isEmpty();
+		return list.noneSatisfy(predicate);
 	}
 }
