--- a/replacePatterns/06-before.java
+++ b/replacePatterns/06-after.java
@@ -1,8 +1,8 @@
 import org.eclipse.collections.api.block.predicate.Predicate;
 import org.eclipse.collections.impl.utility.ArrayIterate;

 class TestArrayIterateGetFirst {
 	String test(String[] array, Predicate<String> predicate) {
-		return ArrayIterate.select(array, predicate).getFirst();
+		return ArrayIterate.detect(array, predicate);
 	}
 }
