--- a/replacePatterns/01-before.java
+++ b/replacePatterns/01-after.java
@@ -1,22 +1,25 @@
-import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;

+import org.eclipse.collections.api.factory.Lists;
+import org.eclipse.collections.api.factory.Maps;
+import org.eclipse.collections.api.factory.Sets;
+
 class Test {
-	private final List<String> fieldSingletonList = Collections.singletonList("element");
-	private final Set<String> fieldSingleton = Collections.singleton("element");
-	private final Map<String, String> fieldSingletonMap = Collections.singletonMap("key", "value");
+	private final List<String> fieldSingletonList = Lists.fixedSize.with("element");
+	private final Set<String> fieldSingleton = Sets.fixedSize.with("element");
+	private final Map<String, String> fieldSingletonMap = Maps.fixedSize.with("key", "value");

 	void test() {
-		List<String> singletonList = Collections.singletonList("element");
-		Set<String> singleton = Collections.singleton("element");
-		Map<String, String> singletonMap = Collections.singletonMap("key", "value");
+		List<String> singletonList = Lists.fixedSize.with("element");
+		Set<String> singleton = Sets.fixedSize.with("element");
+		Map<String, String> singletonMap = Maps.fixedSize.with("key", "value");
 	}

 	void testWithExplicitGenerics(Object element, Object key, Object value) {
-		List<String> singletonList = Collections.<String>singletonList((String) element);
-		Set<String> singleton = Collections.<String>singleton((String) element);
-		Map<String, String> singletonMap = Collections.<String, String>singletonMap((String) key, (String) value);
+		List<String> singletonList = Lists.fixedSize.with((String) element);
+		Set<String> singleton = Sets.fixedSize.with((String) element);
+		Map<String, String> singletonMap = Maps.fixedSize.with((String) key, (String) value);
 	}
 }
