--- a/replacePatterns/04-before.java
+++ b/replacePatterns/04-after.java
@@ -1,8 +1,8 @@
 import org.eclipse.collections.api.block.predicate.Predicate;
 import org.eclipse.collections.impl.utility.ArrayIterate;

 class TestArrayIterate {
 	boolean test(String[] array, Predicate<String> predicate) {
-		return ArrayIterate.select(array, predicate).notEmpty();
+		return ArrayIterate.anySatisfy(array, predicate);
 	}
 }
