--- a/replacePatterns/01-before.java
+++ b/replacePatterns/01-after.java
@@ -1,23 +1,24 @@
 import java.util.Comparator;
 import java.util.List;
 import java.util.Map;
 import java.util.SortedMap;
-import java.util.TreeMap;

+import org.eclipse.collections.api.factory.SortedMaps;
+
 class Test {
-	private final SortedMap<String, String> fieldInterfaceEmpty = new TreeMap<>();
-	private final SortedMap<String, String> fieldInterfaceComparator = new TreeMap<>(Comparator.naturalOrder());
-	private final SortedMap<String, String> fieldInterfaceMap = new TreeMap<>(this.fieldInterfaceEmpty);
+	private final SortedMap<String, String> fieldInterfaceEmpty = SortedMaps.mutable.empty();
+	private final SortedMap<String, String> fieldInterfaceComparator = SortedMaps.mutable.with(Comparator.naturalOrder());
+	private final SortedMap<String, String> fieldInterfaceMap = SortedMaps.mutable.withSortedMap(this.fieldInterfaceEmpty);

 	void test(SortedMap<String, String> inputMap) {
-		SortedMap<String, Integer> typeInference = new TreeMap<>();
-		SortedMap<String, List<Integer>> nestedGenerics = new TreeMap<>();
-		SortedMap<String, ? extends Number> wildcardGenerics = new TreeMap<>();
-		SortedMap<String, Integer> explicitSimple = new TreeMap<String, Integer>();
-		SortedMap<String, List<Integer>> explicitNested = new TreeMap<String, List<Integer>>();
-		java.util.SortedMap<String, Integer> fullyQualified = new TreeMap<>();
-		SortedMap<String, String> sortedMapWithComparator = new TreeMap<>(Comparator.naturalOrder());
-		SortedMap<String, String> interfaceFromMap = new TreeMap<>(inputMap);
-		SortedMap<String, String> fromMap = new TreeMap<>(this.fieldInterfaceEmpty);
+		SortedMap<String, Integer> typeInference = SortedMaps.mutable.empty();
+		SortedMap<String, List<Integer>> nestedGenerics = SortedMaps.mutable.empty();
+		SortedMap<String, ? extends Number> wildcardGenerics = SortedMaps.mutable.empty();
+		SortedMap<String, Integer> explicitSimple = SortedMaps.mutable.<String, Integer>empty();
+		SortedMap<String, List<Integer>> explicitNested = SortedMaps.mutable.<String, List<Integer>>empty();
+		java.util.SortedMap<String, Integer> fullyQualified = SortedMaps.mutable.empty();
+		SortedMap<String, String> sortedMapWithComparator = SortedMaps.mutable.with(Comparator.naturalOrder());
+		SortedMap<String, String> interfaceFromMap = SortedMaps.mutable.withSortedMap(inputMap);
+		SortedMap<String, String> fromMap = SortedMaps.mutable.withSortedMap(this.fieldInterfaceEmpty);
 	}
 }
