--- a/transformsVerifyAssertEmpty/01-before.java
+++ b/transformsVerifyAssertEmpty/01-after.java
@@ -1,17 +1,18 @@
-import org.eclipse.collections.impl.test.Verify;
 import org.eclipse.collections.api.list.MutableList;
 import org.eclipse.collections.impl.factory.Lists;
 import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
 import java.util.HashMap;

 class Test {
 	void test() {
 		MutableList<String> list = Lists.mutable.empty();
-		Verify.assertEmpty("list should be empty", list);
-		Verify.assertEmpty(list);
+		assertThat(list).as("list should be empty").isEmpty();
+		assertThat(list).isEmpty();

 		Map<String, Integer> map = new HashMap<>();
-		Verify.assertEmpty("map should be empty", map);
-		Verify.assertEmpty(map);
+		assertThat(map).as("map should be empty").isEmpty();
+		assertThat(map).isEmpty();
 	}
 }
