--- a/appliesStaticImportOptimization/01-before.java
+++ b/appliesStaticImportOptimization/01-after.java
@@ -1,11 +1,12 @@
-import org.eclipse.collections.impl.test.Verify;
 import org.eclipse.collections.api.list.MutableList;
 import org.eclipse.collections.impl.factory.Lists;

+import static org.assertj.core.api.Assertions.assertThat;
+
 class Test {
 	void test() {
 		MutableList<String> list = Lists.mutable.with("a", "b", "c");
-		Verify.assertSize(3, list);
-		Verify.assertNotEmpty(list);
+		assertThat(list).hasSize(3);
+		assertThat(list).isNotEmpty();
 	}
 }
