--- a/replacePatterns/01-before.java
+++ b/replacePatterns/01-after.java
@@ -1,18 +1,18 @@
-import java.util.SortedSet;
 import java.util.List;
 import org.eclipse.collections.api.factory.SortedSets;
+import org.eclipse.collections.api.set.sorted.MutableSortedSet;
 import org.eclipse.collections.impl.set.sorted.mutable.TreeSortedSet;

 class Test {
-	private final SortedSet<String> fieldSortedSet = SortedSets.mutable.empty();
+	private final MutableSortedSet<String> fieldSortedSet = SortedSets.mutable.empty();

 	void test() {
-		SortedSet<String> simpleSortedSet = SortedSets.mutable.empty();
-		java.util.SortedSet<String> fullyQualifiedSortedSet = SortedSets.mutable.empty();
-		SortedSet rawSortedSet = SortedSets.mutable.empty();
-		java.util.SortedSet rawSortedSetFullyQualified = SortedSets.mutable.empty();
-		SortedSet<List<Integer>> nestedGenerics = SortedSets.mutable.empty();
-		SortedSet<String> treeSortedSet = TreeSortedSet.newSet();
-		SortedSet<String> set1 = SortedSets.mutable.empty(), set2 = SortedSets.mutable.with("a");
+		MutableSortedSet<String> simpleSortedSet = SortedSets.mutable.empty();
+		MutableSortedSet<String> fullyQualifiedSortedSet = SortedSets.mutable.empty();
+		MutableSortedSet rawSortedSet = SortedSets.mutable.empty();
+		MutableSortedSet rawSortedSetFullyQualified = SortedSets.mutable.empty();
+		MutableSortedSet<List<Integer>> nestedGenerics = SortedSets.mutable.empty();
+		MutableSortedSet<String> treeSortedSet = TreeSortedSet.newSet();
+		MutableSortedSet<String> set1 = SortedSets.mutable.empty(), set2 = SortedSets.mutable.with("a");
 	}
 }
