--- a/replacePatterns/01-before.java
+++ b/replacePatterns/01-after.java
@@ -1,17 +1,16 @@
 import org.eclipse.collections.api.list.MutableList;
-import org.eclipse.collections.impl.block.factory.Predicates;
 import org.eclipse.collections.api.block.predicate.Predicate;

 class Test {
 	void test(MutableList<String> list, Predicate<String> predicate) {
-		boolean anySatisfyPredicatesNot = list.anySatisfy(Predicates.not(predicate));
-		boolean anySatisfyPredicatesNotLambda = list.anySatisfy(Predicates.not(s -> s.length() > 5));
-		boolean anySatisfyPredicatesNotMethodRef = list.anySatisfy(Predicates.not(String::isEmpty));
+		boolean anySatisfyPredicatesNot = list.noneSatisfy(predicate);
+		boolean anySatisfyPredicatesNotLambda = list.noneSatisfy(s -> s.length() > 5);
+		boolean anySatisfyPredicatesNotMethodRef = list.noneSatisfy(String::isEmpty);

-		if (list.anySatisfy(Predicates.not(s -> s.isEmpty()))) {
+		if (list.noneSatisfy(s -> s.isEmpty())) {
 			this.doWork();
 		}
 	}

 	void doWork() {}
 }
